app/views/appointments/new.html.erb

```diff
@@ -1,181 +1,218 @@
-            <p class="SlotPicker-date t-slot-2-date"><%= @appointment_form.secondary_slot.formatted_date %></p>
-            <p class="SlotPicker-time t-slot-2-period"><%= @appointment_form.secondary_slot.period %></p>
+        <div class="SlotPicker-choice is-chosen">
+          <div class="SlotPicker-choiceInner">
+            <div class="SlotPicker-position"><span>2</span></div>
+            <div class="SlotPicker-choiceContent">
+              <p class="SlotPicker-date t-slot-2-date"><%= @appointment_form.secondary_slot.formatted_date %></p>
+              <p class="SlotPicker-time t-slot-2-period"><%= @appointment_form.secondary_slot.period %></p>
+            </div>
           </div>
         </div>
-      </div>
 
-      <div class="SlotPicker-choice is-chosen">
-        <div class="SlotPicker-choiceInner">
-          <div class="SlotPicker-position"><span>3</span></div>
-          <div class="SlotPicker-choiceContent">
-            <p class="SlotPicker-date t-slot-3-date"><%= @appointment_form.tertiary_slot.formatted_date %></p>
-            <p class="SlotPicker-time t-slot-3-period"><%= @appointment_form.tertiary_slot.period %></p>
+        <div class="SlotPicker-choice is-chosen">
+          <div class="SlotPicker-choiceInner">
+            <div class="SlotPicker-position"><span>3</span></div>
+            <div class="SlotPicker-choiceContent">
+              <p class="SlotPicker-date t-slot-3-date"><%= @appointment_form.tertiary_slot.formatted_date %></p>
+              <p class="SlotPicker-time t-slot-3-period"><%= @appointment_form.tertiary_slot.period %></p>
+            </div>
           </div>
         </div>
       </div>
-    </div>
 
-    <h3 class="h4">Requested location:</h3>
-    <p class="lead">
-      <strong>
-        <a href="https://www.pensionwise.gov.uk/locations/<%= @appointment_form.location_aware_booking_request.location_id %>" class="t-location-name">
-          <%= guard_missing_location(@appointment_form.location_aware_booking_request, :location_name) %>
-        </a>
-      </strong>
-    </p>
-    <hr>
-    <%= toggle_activation_link(@appointment_form.location_aware_booking_request) do |action| %>
-      <span aria-hidden="true" class="glyphicon glyphicon-eye-<%= action == 'hide' ? 'close' : 'open' %>"></span>
-      <%= action.capitalize %> this Booking Request
-    <% end %>
-  </div>
+      <h3 class="h4">Requested location:</h3>
+      <p class="lead">
+        <strong>
+          <a href="https://www.pensionwise.gov.uk/locations/<%= @appointment_form.location_aware_booking_request.location_id %>" class="t-location-name">
+            <%= guard_missing_location(@appointment_form.location_aware_booking_request, :location_name) %>
+          </a>
+        </strong>
+      </p>
+      <hr>
+      <button type="button" class="btn btn-default btn-block booking-request-status-btn js-booking-request-change-state-button t-booking-request-change-state-button">
+        Change booking request state
```

Comment:
Ah yes, good call, probably the result of some copypasta from the appointments form.